### PR TITLE
chore: node20

### PR DESCRIPTION
**Pull request by nipe0324**
I tried to update nod20 from node16

- #38

### CODE REVIEW

**Comment by aki77** [comment]
Thanks!

File: package.json [hunk 1]
```diff
@@ -33,9 +33,9 @@
   "devDependencies": {
     "@types/jest": "^27.0.2",
     "@types/markdown-table": "^3.0.0",
-    "@types/node": "^16",
+    "@types/node": "^20",
     "@typescript-eslint/parser": "^5.16.0",
-    "@zeit/ncc": "^0.22.3",
+    "@vercel/ncc": "^0.38.1",
```
Comment (nipe0324) [inline package.json:38]: ncc build is failed.
https://github.com/aki77/simplecov-report-action/actions/runs/7824831804/job/21356571624

so, I use `vercel/ncc` as a refarence 
https://github.com/actions/typescript-action/blob/d88c8226076df14de5a25dfeb0a4961ad04a9084/package.json#L76

**Comment by nipe0324** [comment]
@aki77 

The reason of `Error: Resource not accessible by integration`  of `build-test / test` may be a workflow permissions issue.
Could you please check the Workflow permissions? 🤔 
https://github.com/orgs/community/discussions/60820

**Comment by nipe0324** [comment]
@aki77 how about this https://github.com/aki77/simplecov-report-action/pull/39#issuecomment-1936839474 ?? 🤔

**Comment by lsilva-justworks-com** [comment]
> @aki77 how about this [#39 (comment)](https://github.com/aki77/simplecov-report-action/pull/39#issuecomment-1936839474) ?? 🤔

It would be nice to have this on node20. I am getting a warning

**Pull request closed without merging**